FRBDK/Glue/Glue/CodeGeneration/ReferencedFileSaveCodeGenerator.cs

```diff
@@ -784,7 +788,7 @@ public static void GetLoadCallForAtiFile(ReferencedFileSave rfs, AssetTypeInfo a
             if (ati != null && !referencedFile.IsCsvOrTreatedAsCsv)
             {
                 GetLoadCallForAtiFile(referencedFile, ati, contentManagerName,
-                    ifBlock, element);
+                    ifBlock, element, null);
```

Comment:
This is what I'm not sure about. There is no `project` here (well, unless we check the whole callchain, probably available at some level) although in my tests this part never gets executed.